kong/db/dao/targets.lua

```diff
@@ -214,7 +214,7 @@ function _TARGETS:page_for_upstream(upstream_pk, size, offset, options)
 
   local next_offset
   if all_active_targets[size + offset + 1] then
-    next_offset = size + offset
+    next_offset = tostring(size + offset)
```

Comment:
In Cassandra, `offset` is a base64-encoded string. Has this change been tested with the C* strategy?

Reply:
Good point. I've tested this only with Postgres manually and had wrongly assumed that the test ran both Postgres and C*. Let me test with C*.

Reply:
Actually I take that back. The tests run using both the strategies and pass.

Reply:
Oh, this endpoint defines its own logic to generate the offset, I see... Well, nowadays in the new DAO, PostgreSQL should have base64-encoded offsets as well, so this endpoint is somewhat of an oddity now...

Reply:
Let's change this to a `string`, indeed. But for one additional reason, which is that all `offset` return values from the DAO are string, and are tested as such (see tests in https://github.com/Kong/kong/blob/next/spec/02-integration/000-new-dao/02-db_core_entities_spec.lua).

We should add a DAO-level test asserting that the return `offset` value is a string (on top of the Admin API one that we can keep as a regression test).